frontend/src/static/css/v_summary.scss

```diff
@@ -1,189 +1,209 @@
 @import 'colors';
 
-/* summary chart */
-#summary,
-.summary-status {
-  text-align: center;
-}
-
-.icon-button {
-  color: mui-color('grey');
-  cursor: pointer;
-  text-decoration: none;
-}
-
-.summary-picker {
-  align-items: center;
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
-  margin-bottom: 2rem;
-
-  &__section {
-    align-items: inherit;
-    display: flex;
-    flex: 0 1 auto;
-    flex-flow: inherit;
-    justify-content: inherit;
-  }
+/* Summary */
+#summary {
+  .fileTypes {
```

Comment:
Is it possible to place your new css selectors at the bottom such that the other components won't be shown with changes.

Reply:
I actually did shift everything 2 space to right cause the original css is not really correct.
```
#summary,
.sth{}
.sth{}
```
So everything below is not under `#summary`

Reply:
Ahh I see. My bad for not spotting that.